src/ui/pages/equipment/asset/Asset.js

```diff
@@ -335,9 +338,38 @@ export default class Asset extends Entity {
                 column: 2,
                 order: 11
             },
+            {
+                id : 'PARTCUSTOMFIELDS',
+                label : 'Part Custom Fields',
+                isVisibleWhenNewEntity: false,
+                maximizable: false,
+                customVisibility: () => this.state.part,
```

Comment:
To be discussed on what conditions to show/hide the custom fields for the part.

Reply:
Discussed with David, the region should not appear when there is no part associated with the asset.